gg-pac-team3-back/src/routes/userCourseRoutes.mjs

```diff
@@ -0,0 +1,13 @@
+import express from 'express';
+import { enrollInCourse, getUserCourses } from '../controllers/userCourseController.mjs';
+import { verifyToken } from '../middlewares/verifyToken.mjs';
+
+const router = express.Router();
+
+// Route to enroll in a course
+router.post('/courses/:courseId/enrollments', verifyToken, enrollInCourse);
```

Comment:
Nit: I would make this `/users/:userId/course` and the one below the same (just different methods).

Reply:
I will do changes. Thank you Eli!